examples/rpc_send_example.c

```diff
@@ -145,8 +144,16 @@ main(int argc, char **argv)
         goto cleanup;
     }
 
+    /* RPC input */
+    sr_val_t *input = NULL;
+    const size_t input_cnt = 0;
```

Comment:
I do not think using such empty structures is an improvement because it may easily be misinterpreted in the way that it is required even though it is not. I would prefer you keep it the way it was.

Reply:
Done